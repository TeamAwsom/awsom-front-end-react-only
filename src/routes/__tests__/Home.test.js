import React from 'react';
import Home from '../Home';
import renderer from 'react-test-renderer';

describe('Home', () => {
  it('renders as expected', () => {
    const tree = renderer.create(<Home />).toJSON();

    console.log(tree);
    expect(tree).toMatchSnapshot();
  });
});
