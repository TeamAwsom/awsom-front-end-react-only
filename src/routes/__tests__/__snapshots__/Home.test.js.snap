// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home renders as expected 1`] = `
<main
  className="container"
>
  <div
    className="main-wrapper"
  >
    <div
      className="intro"
    >
      <h1>
        Making music. Your home. Your time.
      </h1>
      <p>
        We are excited that you are looking into the journey of learning music with us. We hire only the highest quality of teachers to assist our students as they embark on the exciting opportunity of starting or continuing with an instrument. Follow the links below to find out more about the instruments we offer.
      </p>
    </div>
    <div
      className="nav-down"
      id="choose-adv"
    >
      <h1
        style={
          Object {
            "opacity": "0",
            "transition": "opacity .5s ease-in",
          }
        }
      >
        Choose Your Instrument
      </h1>
    </div>
    <div
      className="wrapper container-fluid"
    >
      <div
        className="row"
      >
        <div
          className="col-sm-5 offset-sm-1"
        >
          <div
            className="cards"
          >
            <a
              href="https://www.awsom.info/piano/"
            >
              <img
                alt="piano"
                src="piano.jpg"
              />
            </a>
            <div
              className="image-caption-wrapper"
            >
              <a
                className="image-caption"
                href="https://www.awsom.info/piano/"
              >
                Piano
              </a>
            </div>
          </div>
        </div>
        <div
          className="col-sm-5 card_container"
        >
          <div
            className="cards"
          >
            <a
              href="https://www.awsom.info/guitar/"
            >
              <img
                alt="guitar"
                src="guitar.jpg"
              />
            </a>
            <div
              className="image-caption-wrapper"
            >
              <a
                className="image-caption"
                href="https://www.awsom.info/guitar/"
              >
                Guitar
              </a>
            </div>
          </div>
        </div>
      </div>
      <div
        className="row"
      >
        <div
          className="col-sm-5 offset-sm-1"
        >
          <div
            className="cards"
          >
            <a
              href="https://www.awsom.info/drums/"
            >
              <img
                alt="drums"
                src="drums.jpg"
              />
            </a>
            <div
              className="image-caption-wrapper"
            >
              <a
                className="image-caption"
                href="https://www.awsom.info/drums/"
              >
                Drums
              </a>
            </div>
          </div>
        </div>
        <div
          className="col-sm-5"
        >
          <div
            className="cards"
          >
            <a
              href="https://www.awsom.info/woodwind/"
            >
              <img
                alt="woodwind"
                src="woodwind.jpg"
              />
            </a>
            <div
              className="image-caption-wrapper"
            >
              <a
                className="image-caption"
                href="https://www.awsom.info/woodwind/"
              >
                Woodwind
              </a>
            </div>
          </div>
        </div>
      </div>
      <div
        className="row"
      >
        <div
          className="col-sm-5 offset-sm-1"
        >
          <div
            className="cards"
          >
            <a
              href="https://www.awsom.info/voice/"
            >
              <img
                alt="voice"
                src="voice.jpg"
              />
            </a>
            <div
              className="image-caption-wrapper"
            >
              <a
                className="image-caption"
                href="https://www.awsom.info/voice/"
              >
                Voice
              </a>
            </div>
          </div>
        </div>
        <div
          className="col-sm-5"
        >
          <div
            className="cards"
          >
            <a
              href="https://www.awsom.info/brass/"
            >
              <img
                alt="brass"
                src="brass.jpg"
              />
            </a>
            <div
              className="image-caption-wrapper"
            >
              <a
                className="image-caption"
                href="https://www.awsom.info/brass/"
              >
                Brass
              </a>
            </div>
          </div>
        </div>
      </div>
      <div
        className="row"
      >
        <div
          className="col-sm-5 offset-sm-1"
        >
          <div
            className="cards"
          >
            <a
              href="https://www.awsom.info/strings/"
            >
              <img
                alt="strings"
                src="strings.jpg"
              />
            </a>
            <div
              className="image-caption-wrapper"
            >
              <a
                className="image-caption"
                href="https://www.awsom.info/strings/"
              >
                Strings
              </a>
            </div>
          </div>
        </div>
        <div
          className="col-sm-5"
        >
          <div
            className="cards"
          >
            <a
              href="https://www.awsom.info/chamber/"
            >
              <img
                alt="chamber"
                src="chamber.jpg"
              />
            </a>
            <div
              className="image-caption-wrapper"
            >
              <a
                className="image-caption"
                href="https://www.awsom.info/chamber/"
              >
                Chamber
              </a>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</main>
`;
